inputfiles/addedTypes.json

```diff
@@ -1437,5 +1437,11 @@
             "name": "reason?",
             "type": "any"
         }]
+    },
+    {
+        "kind": "method",
+        "interface": "Body",
```

Comment:
should have ` "flavor": "Web",` to avoid adding `FormData` in webworkers.d.ts, DOM is not available in web workers.